views/pages/ConfigurationsPage/index.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react'
+import { Box, Button } from '@mui/material'
+import { Add } from '@mui/icons-material'
+import { useOutletContext, Link } from 'react-router-dom'
+
+import ConfigurationsTable from '../../tables/ConfigurationsTable'
+import PageHeader from '../../components/PageHeader'
+import { UsersModel } from '../../models'
+import api from '../../api'
+import ShareConfigurationForm from '../../forms/ShareConfigurationForm'
+import { routes } from '../../routes/routes'
+
+const ConfigurationsPage = () => {
+  const {
+    configurations,
+    navigate,
+    setNotification,
+    setConfigurations,
+    setUser,
+    user,
+    users,
+  } = useOutletContext()
+  const { uid, preferences } = user
+  const [configuration, setConfiguration] = useState(null)
+  const open = !!configuration?._id
+  const options =
+    configuration?.readers && configuration?.owner
+      ? UsersModel.createUserFriendList(users, configuration.owner)
+      : []
+  const formValues =
+    configuration?.readers && configuration?.owner
+      ? UsersModel.createOptionsFromReaders(
+          configuration.readers,
+          configuration.owner
+        )
+      : []
```

Comment:
related to the comment below about passing the initial values in the shape the form expects.

```suggestion
  const initialValues =
    configuration?.readers && configuration?.owner
      ? { readers: UsersModel.createOptionsFromReaders(configuration.readers, configuration.owner) }
      : { readers: [] }
```

Reply:
[47a7352](https://github.com/AMP-SCZ/dpdash/pull/487/commits/47a7352aa90256c68d3515ee70b8a513b3052989)